app/controllers/shelters_controller.rb

```diff
@@ -43,4 +43,9 @@ def update
     shelter.save
     redirect_to "/shelters/#{shelter.id}"
   end
+
+  def pets
+    @shelter = Shelter.find(params[:id])
+    @shelter_pets = @shelter.pets
+  end
```

Comment:
I was hesitant to put pets in the shelter controller because now maybe the shelter controller is not encapsulated. I am wondering if it would be best practice to make a shelter_pets controller with its own index.